rest_framework/validators.py

```diff
@@ -211,7 +221,9 @@ def __call__(self, attrs):
         queryset = self.exclude_current_instance(attrs, queryset)
         if queryset.exists():
             message = self.message.format(date_field=self.date_field)
-            raise ValidationError({self.field: message})
+            raise ValidationError({
+                self.field: ValidationErrorMessage(message, code='unique'),
```

Comment:
Can you explain how the error argument for nested cases changes here ? I'm not following that part.

From what I get, everything stayed the same, it's just that the `message` became a little bit smarter than before and it is opt-in so you'd only have to change anything if you actually want to use error codes.